bin/k8s-deploy

```diff
@@ -6,31 +6,31 @@ echo "Deploying ConfigMaps"
 for i in "${CONFIGMAP_FILES[@]}"
 do
   echo "Creating $i"
-  kubectl create -f $i
+  kubectl apply -f $i
 done
 echo "Done deploying ConfigMaps"
 
 echo "Deploying Secrets"
 for i in "${SECRET_FILES[@]}"
 do
   echo "Creating $i"
-  kubectl create -f $i
+  kubectl apply -f $i
 done
 echo "Done deploying Secrets"
 
 echo "Deploying Services"
 for i in "${SERVICE_FILES[@]}"
 do
   echo "Creating $i"
-  kubectl create -f $i
+  kubectl apply -f $i
 done
 echo "Done deploying Services"
 
 echo "Deploying Deployments"
 for i in "${DEPLOYMENT_FILES[@]}"
 do
   echo "Creating $i"
-  kubectl create -f $i
+  kubectl apply -f $i
```

Comment:
I switched from "kubectl create" to "kubectl apply" since creation fails if the resource already exists, and they typically do in these cases. I know we discussed it, and I'm open to alternatives.